Dockerfile.test.gpu

```diff
@@ -96,7 +96,11 @@ RUN if [[ ${PYTORCH_PACKAGE} == "torch-nightly" ]]; then \
 RUN pip install "Pillow<7.0" --no-deps
 
 # Install MXNet.
-RUN pip install ${MXNET_PACKAGE}
+RUN if [[ ${MXNET_PACKAGE} == "mxnet-nightly" ]]; then \
+        pip install -pre mxnet_cu100 -f https://dist.mxnet.io/python/all; \
```

Comment:
Tests for the nightly builds are running on CUDA 10.1.  Should this be `mxnet_cu101`?

Reply:
Updated to cu101